pontoon/base/templates/widgets/latest_activity.html

```diff
@@ -1,12 +1,16 @@
-{% macro span(latest_activity) -%}
+{% macro span(latest_activity, resource='all-resources') -%}
 <span class="latest">
   {% if latest_activity.date %}
-  <time datetime="{{ latest_activity.date.isoformat() }}">{{ latest_activity.date|naturaltime }}</time>
     {% if latest_activity.user %}
-    &bull; <a href="{{ url('pontoon.contributor.username', latest_activity.user.username) }}">{{ latest_activity.user.name_or_email|nospam }}</a>
+      {% set action = latest_activity.type + ' by ' + latest_activity.user.name_or_email %}
+    {% else %}
+      {% set action = 'imported' %}
     {% endif %}
+    <a href="{{ url('pontoon.translate', latest_activity.translation.locale.code, latest_activity.translation.entity.resource.project.slug, resource) }}?string={{ latest_activity.translation.entity.id }}" title="Translation '{{ latest_activity.translation.string }}' {{action}}">
```

Comment:
I noticed that `/teams/` view started to take 30 seconds to load on my local machine. After I commented out `latest_activity.html`, loading time dropped to 10 seconds. I wonder if we should prefetch these relations or even denormalize last activity.